### DISCUSSION TITLE
HLK-LD2410S only on ESP32?

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#22496] [Q&A]
Harware ESP2866 (Shelly )?

i compiled with 
```
#define USE_LD2410S
#define USE_LD2410

```

but in config only `LD2410 Rx` and `LD2410 Tx` is shown up.

**Comment by sfromis** [comment]
For some reason, both drivers have the same number, `xsns_102`, which seems to mean that you can't have both included at the same time.

**Reply by ChrisVonFrosta** [reply]
Thank you! I was just about to write that the LD2410 (without the S) also doesn't work, but then I'll try it first with individual drivers.
